docs/CONTRIBUTING.md

```diff
@@ -56,18 +56,41 @@ Vector team member will find this document useful.
 
 ### New sources, sinks, and transforms
 
-If you're contributing a new source, sink, or transform to Vector, thank you that's way cool! There's a few steps you
-need to think about if you want to make sure we can merge your contribution. We're here to help you along with these steps,
-but they are a blocker to getting a new integration released.
-
-To merge a new source, sink, or transform, you need to:
+If you're thinking of contributing a new source, sink, or transform to Vector, thank you that's way cool! There's a few
+steps you need to think about if you want to make sure we can merge your contribution. We're here to help you along with
+these steps, but they are a blocker to getting a new integration released.
+
+Prior to beginning work on a new source or sink if a GitHub Issue does not already exist, please open one to discuss the
+introduction of the new integration. Maintainers will review the proposal with the following checklist in mind, try and
```

Comment:
I feel like the first sentence should be bolded

Reply:
Addressed in [bbc18b2](https://github.com/vectordotdev/vector/pull/17658/commits/bbc18b285de51e66cd01fcd1c5f89efc71bcfff1)